lib/virt_autotest/utils.pm

```diff
@@ -499,11 +498,11 @@ sub create_guest {
         my ($autoyastURL, $diskformat, $virtinstall);
         $autoyastURL = $autoyast;
         $diskformat = get_var("VIRT_QEMU_DISK_FORMAT") // "qcow2";
-        $extra_args = "$linuxrc autoyast=$autoyastURL $extra_args";
+        $extra_args = "autoyast=$autoyastURL $extra_args";
         $extra_args = trim($extra_args);
         $virtinstall = "virt-install $v_type $guest->{osinfo} --name $name --vcpus=$vcpus,maxvcpus=$maxvcpus --memory=$memory,maxmemory=$maxmemory --vnc";
         $virtinstall .= " --disk path=/var/lib/libvirt/images/$name.$diskformat,size=20,format=$diskformat --noautoconsole";
-        $virtinstall .= " --network network=default,mac=$macaddress --autostart --location=$location --wait -1";
+        $virtinstall .= " --network bridge=br0 --autostart --location=$location --wait -1";
```

Comment:
br0 will become a necessary host setup, right? I do not see this part set in host autoyast file or test code. Do all products sle12sp3->15sp6 have br0 by default?

Reply:
For [sle12](https://github.com/tbaev/os-autoinst-distri-opensuse/blob/e2273fd8f6941b64ddb719b21965a96bf3da40cc/data/virtualization/autoyast/host_12.xml.ep#L114) and [sle15](https://github.com/tbaev/os-autoinst-distri-opensuse/blob/e2273fd8f6941b64ddb719b21965a96bf3da40cc/data/virtualization/autoyast/host_15.xml.ep#L145) hosts, the br0 is define before my changes.

```
  <interface>
    <device>br0</device>
    <bootproto>dhcp</bootproto>
    <bridge>yes</bridge>
    <bridge_forwarddelay>0</bridge_forwarddelay>
    <bridge_ports>eth0 eth1</bridge_ports>
    <bridge_stp>off</bridge_stp>
    <startmode>auto</startmode>
  </interface>
```

Reply:
Oh, sorry for overlooking that. Then it will be fine.